scanner/config/config_test.go

```diff
@@ -161,14 +159,11 @@ func Test_MatcherConfig_validate(t *testing.T) {
 		assert.False(t, c.RemoteIndexerEnabled)
 	})
 	t.Run("when URL is replaceable, replace it", func(t *testing.T) {
-		c := MatcherConfig{Enable: true, Database: Database{ConnString: "host=foobar"}, VulnerabilitiesURL: "https://central.stackrox.svc/api/extensions/scannerdefinitions?version=ROX_VERSION"}
+		c := MatcherConfig{Enable: true, Database: Database{ConnString: "host=foobar"}, VulnerabilitiesURL: "https://central.stackrox.svc/api/extensions/scannerdefinitions?rox_version=ROX_VERSION&vuln_version=ROX_VULNERABILITY_VERSION"}
```

Comment:
Is the plan to change `version` to `rox_version`? If so, do we plan on continuing to support `version` to account for (hopefully temporary) version mismatches?

Reply:
Nah, this is just a way to be verbose in the test. The default URL config will continue to use `version`; We will only move way from `ROX_VERSION` into `ROX_VULNERSBILITY_VERSION` in the deployments. From the definition handler perspective, this will always try to get the specified version -- either release, schema or PR names, or whatever.